docs/unique-design-challenges.md

```diff
@@ -76,4 +77,25 @@ While not part of the core nostr protocol, zaps are popular enough that we shoul
 
 In this guide we will explore how current clients are explaining zaps (if they are) and where / how we can improve the overall user experience by doing a better job at arming the user with the necessary information.
 
-[Explore Zaps](https://nostrdesign.org/docs/how-to/zaps/)
\ No newline at end of file
+[Explore Zaps](https://nostrdesign.org/docs/how-to/zaps/)
+
+## Challenge #5: NIPs
+
+Unlike email clients (e.g. yahoo email client, microsoft outlook email client, gmail email client etc.) which (explaining in an oversimplified way) either support or don't support the email protocol, nostr apps have tens of NIPs, and potentially hundreds of NIP candidates as nostr continues to grow. 
+
+A design consequence of a multitude of opt-in NIPs means that there will be sets of apps with little overlap in supported NIPs. This means that a user of app X may not be able to interact with a user on app Y, or vice versa. This also means that unless this is surfaced to either users, they may not be aware that they cannot interact with users of another app. 
```

Comment:
I would avoid using X because it could be misunderstood for “Twitter", first of all in this context; A/B seems a better choice.